src/command/feed/print.ts

```diff
@@ -26,26 +27,39 @@ export class Print extends FeedCommand implements LeafCommand {
     await super.init()
 
     const topic = this.topic || this.bee.makeFeedTopic(this.topicString)
-    this.console.info('Looking up feed topic ' + topic + '...')
-    const addressString = this.address || (await this.getAddressString())
-    const reader = this.bee.makeFeedReader('sequence', topic, addressString)
-    const { reference, feedIndex, feedIndexNext } = await reader.download()
+    const spinner = createSpinner('Looking up feed topic ' + topic)
+    spinner.start()
+    try {
+      const addressString = this.address || (await this.getAddressString())
+      const reader = this.bee.makeFeedReader('sequence', topic, addressString)
+      const { reference, feedIndex, feedIndexNext } = await reader.download()
+
+      if (!this.stamp) {
+        this.stamp = await pickStamp(this.beeDebug, this.console)
+      }
 
-    if (!this.stamp) {
-      this.stamp = await pickStamp(this.beeDebug, this.console)
+      const { reference: manifest } = await this.bee.createFeedManifest(this.stamp, 'sequence', topic, addressString)
+
+      spinner.stop()
+      this.console.verbose(createKeyValue('Chunk Reference', reference))
+      this.console.verbose(createKeyValue('Chunk Reference URL', `${this.bee.url}/bzz/${reference}/`))
+      this.console.verbose(createKeyValue('Feed Index', feedIndex))
+      this.console.verbose(createKeyValue('Next Index', feedIndexNext))
+      this.console.verbose(createKeyValue('Feed Manifest', manifest))
+
+      this.console.quiet(manifest)
+      this.console.log(createKeyValue('Feed Manifest URL', `${this.bee.url}/bzz/${manifest}/`))
+      this.console.log(createKeyValue('Number of Updates', parseInt(feedIndex, 10) + 1))
+    } catch (ex: any) {
+      spinner.stop()
+      this.console.info('Feed topic lookup error:')
+      this.console.error(`Status: ${ex.response.status} Message: ${ex.response.statusText}`)
+      this.console.error(ex.message)
+    } finally {
+      if (spinner.isSpinning) {
+        spinner.stop()
+      }
```

Comment:
No need to wrap it in the condition, you can keep it a one liner with only `spinner.stop()`